### PR TITLE
NEPT-597: Remove _ui modules from hard dependencies.

### PR DESCRIPTION
**Pull request by drishu**
### Description

Remove _ui modules from hard dependencies and added them back in soft config.

### Change log

- Removed: UI modules dependencies from info files.

### CODE REVIEW

File: profiles/common/modules/features/cce_basic_config/cce_basic_config.install [hunk 1]
```diff
@@ -82,6 +82,22 @@ function cce_basic_config_requirements($phase) {
...
+    'og_ui',
+    'print_ui',
+    'rules_admin',
+    'tmgmt_ui',
+    'tmgmt_entity_ui',
+    'views_ui',
+  );
+  module_enable($modules);
+
   // Create contributor editor roles.
```
Comment (kameleon69) [inline profiles/common/modules/features/cce_basic_config/cce_basic_config.install]: I discussed with Delphine Lepers and you should not activate the module in the installation.

**Review by tomv-be — DISMISSED** [review]
Ok for me.

**Review by kameleon69 — DISMISSED** [review]
ok

**Review by kameleon69 — APPROVED** [review]
Fine for me.

**Pull request merged**